live_flights_on_map.md

```diff
@@ -0,0 +1,125 @@
+## Documentation for live flight details on a map
+
+##### 1. Map Creation
+
+Firstly, we have to insert OpenLayers css into an HTML file:
```

Comment:
@arnabdutta73 Can you explain the thought process behind choosing openlayers here? this will help the readers to get some understanding about openlayers and your decision.

Reply:
Done